libs/community/langchain_community/utilities/brave_search.py

```diff
@@ -48,8 +50,8 @@ def download_documents(self, query: str) -> List[Document]:
         results = self._search_request(query)
         return [
             Document(
-                page_content=item.get("description"),  # type: ignore[arg-type]
-                metadata={"title": item.get("title"), "link": item.get("url")},
+                page_content=item.get("extra_snippets"),  # type: ignore[arg-type]
+                metadata={"title": item.get("title"), "link": item.get("url"), "description": item.get("description")},
```

Comment:
Please consult our [contributing guide](https://python.langchain.com/docs/contributing/code) for instructions here (you may need to run `make format-diff`).